### PR TITLE
Improvements to Order Confirmation

### PR DESCRIPTION
**Pull request by alexandersanford**
Some fixes to ensure correct processing and confirming of order, sends email to confirm order submission. Working on improving the formatting of date format to match orders from the CSV simulation file.

### CODE REVIEW

File: src/main/java/com/dreamteam/database/EmailService.java [hunk 1]
```diff
@@ -90,125 +85,160 @@ public static void SendMessage() throws MessagingException
...
 				}
 				
-				emailFolder.close(false); //TODO Should this be hardcoded?
-				store.close();
-			}
-		catch (NoSuchProviderException e)
-			{
-				e.printStackTrace();
-				orderContents = new String[] {""};
+				//if(message2.getSubject().contains("order"))
```
Comment (JonathanGrant92) [inline src/main/java/com/dreamteam/database/EmailService.java:140]: Remove commented if we don't need it anymore.

Comment (JonathanGrant92) [inline src/main/java/com/dreamteam/database/EmailService.java:140]: Remove commented if we don't need it anymore.

**Review by JonathanGrant92 — CHANGES_REQUESTED** [review]

**Review by JonathanGrant92 — APPROVED** [review]

**Review by kevvo15 — APPROVED** [review]
After resolving conflicts, it appears that everything should function normally. Great job!

**Comment by JonathanGrant92** [comment]
Resolves #80

**Pull request merged**